src/mpid/ch3/include/mpidpost.h

```diff
@@ -169,7 +169,12 @@ int MPIDI_CH3_Comm_connect(char * port_name, int root, MPIR_Comm * comm_ptr,
 #define MPID_Progress_start(progress_state_) MPIDI_CH3_Progress_start(progress_state_)
 #define MPID_Progress_wait(progress_state_)  MPIDI_CH3_Progress_wait(progress_state_)
 #define MPID_Progress_end(progress_state_)   MPIDI_CH3_Progress_end(progress_state_)
-#define MPID_Progress_test()                 MPIDI_CH3_Progress_test()
+/* This is static inline instead of macro because otherwise MPID_Progress_test will
```

Comment:
Can we make this a separate commit, as well?

Reply:
Done